pkg/controllers/ttl/controller.go

```diff
@@ -117,6 +118,34 @@ func (c *controller) deregisterEventHandlers() {
 	c.logger.V(3).Info("deregistered event handlers")
 }
 
+// Function to determine the deletion propagation policy
+func (c *controller) determinePropagationPolicy(metaObj metav1.Object, logger logr.Logger) *metav1.DeletionPropagation {
+	annotations := metaObj.GetAnnotations()
+	var policy *metav1.DeletionPropagation
+
+	if annotations != nil {
+		annotationPolicy := annotations["kyverno.AnnotationCleanupPropagationPolicy"]
```

Comment:
`kyverno.AnnotationCleanupPropagationPolicy` is not a string literal

Reply:
Okay I removed the double quotes.